query/src/lib.rs

```diff
@@ -50,11 +46,12 @@ pub enum SrcVar {
 }
 
 /// These are the scalar values representable in EDN.
-#[derive(Clone,Debug,Eq,PartialEq)]
+// Floats and doubles only implement PartialEq, but not Eq.
+#[derive(Clone,Debug,PartialEq)]
 pub enum NonIntegerConstant {
     Boolean(bool),
-    BigInteger(BigInt),
-    Float(OrderedFloat<f64>),
+    BigInteger(String),
```

Comment:
I'll need this as numeric, I think; `query` (and `query_parser`, which hasn't merged yet) need to pass along `BigInt`. I don't think there's a Rust/Cargo way of doing that neatly.